external-crates/move/crates/move-binary-format/src/file_format_common.rs

```diff
@@ -409,8 +446,12 @@ pub const VERSION_5: u32 = 5;
 ///  + u16, u32, u256 integers and corresponding Ld, Cast bytecodes
 pub const VERSION_6: u32 = 6;
 
+/// Version 7: changes compared with version 6
+///  + enums
+pub const VERSION_7: u32 = 7;
+
 // Mark which version is the latest version
-pub const VERSION_MAX: u32 = VERSION_6;
+pub const VERSION_MAX: u32 = VERSION_7;
```

Comment:
just gotta be extra careful here. For ny own sanity, can you point to where we check this setting?

Reply:
Not sure if I follow fully here -- do you mean in the serializer and deserializer?

Reply:
Where we check it on the sui side

Reply:
So checked through, and the only time `VERSION_MAX` is used anywhere is on read-path only code. 

Otherwise, the `move_binary_format` version is used from the protocol config everywhere, e.g., when deserializing the module.